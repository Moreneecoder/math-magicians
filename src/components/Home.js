import '../stylesheets/home.css';
import Math from '../images/math.svg';

const Home = () => (
  <div className="Home">
    <img className="mathImg" src={Math} alt="math" />
    <h1 className="center-align">Where The Magic Of Math Is Stripped Bare</h1>

    <div className="desc">
      <p>
        Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor
        incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud
        exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute
        irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat
        nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in
        culpa qui o!cia deserunt mollit anim id est laborum. Ipsu
      </p>

      <p>
        Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor
        incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud
        exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute
        irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat
        nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in
        culpa qui o!cia deserunt mollit anim id est laborum. Ipsu
      </p>
    </div>

  </div>
);

export default Home;
