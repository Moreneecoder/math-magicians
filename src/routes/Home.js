const Home = () => (
  <div className="Home">
    <h1>No Place Like Home</h1>
  </div>
);

export default Home;
