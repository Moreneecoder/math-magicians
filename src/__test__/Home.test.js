// import { render, screen } from "@testing-library/react"
import renderer from 'react-test-renderer';
import Home from '../components/Home'

// test(`on render, page should contain h1 of 'Where Magic Of Maths Is Stripped Bare'`, () => {
//     render(<Home />)

//     // screen.debug();
//     expect(screen.getByRole('heading',
//     {name: /Where The Magic Of Math Is Stripped Bare/}))
//     .toBeInTheDocument()

// })

describe('Home Component', () => {
    test('Renders home component as intended', () => {
      const tree = renderer.create(<Home />).toJSON();
      expect(tree).toMatchSnapshot();
    });
});