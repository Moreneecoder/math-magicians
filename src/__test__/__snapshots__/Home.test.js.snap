// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Component Renders home component as intended 1`] = `
<div
  className="Home"
>
  <img
    alt="math"
    className="mathImg"
    src="math.svg"
  />
  <h1
    className="center-align"
  >
    Where The Magic Of Math Is Stripped Bare
  </h1>
  <div
    className="desc main-bg-color"
  >
    <p>
      Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui o!cia deserunt mollit anim id est laborum. Ipsu
    </p>
  </div>
</div>
`;
